gradle-version/test/gradle-version-test.coffee

```diff
@@ -0,0 +1,63 @@
+Helper = require('hubot-test-helper')
+chai = require 'chai'
+nock = require('nock')
```

Comment:
Did you follow an article for how to mock HTTP requests? The one I followed used a library called `co` to simply add delay to the live call, and I couldn't get it to work with Coffeescript (it needed a reference to `this` passed to it)

Reply:
Yepp:
https://hubot.github.com/docs/scripting/#testing-hubot-scripts
https://www.npmjs.com/package/nock
https://amussey.github.io/2015/08/11/testing-hubot-scripts.html